src/utils/probe-utils.js

```diff
@@ -12,14 +10,6 @@ export function sortByKey(key) {
   };
 }
 
-export const sortByHistogramObjectKey = (a, b) => {
-  const ai = +a;
-  const bi = +b;
-  if (ai < bi) return -1;
-  if (ai >= bi) return 1;
-  return 0;
-};
-
 export const responseHistogramToGraphicFormat = (
```

Comment:
Also not part of this PR, really, but do we need to export this? I don't see it being used externally.